### ISSUE TITLE
Move to using android-json means that JSONException is now checked

### ISSUE THREAD

**Issue by wilkinsona** [skyscreamer/JSONassert#73]
The change made in https://github.com/skyscreamer/JSONassert/commit/2f3576c72a28db3292740e5c67ffbc856394af96 means that the JSONException that's pulled in transitively is now a checked exception. As a result, code that compiled against JSONassert 1.3 does not compile against 1.4.

**Comment by bitkid** [comment]
yeah thats super ugly .. they should be wrapped into a runtime exception
